File: process-fsharp/process_test.fs
module ProcessFSharpTests

open NUnit.Framework
open FsUnit
open ProcessFSharp

[<TestFixture>]
type ProcessFSharpTest() =

    [<Test>]
    member this.ShouldWork() =
        processFSharp "Name" "../../test_data" |> should equal SUCCESS

File: process-csharp/process_test.cs
using System;
using System.Collections.Generic;
using NUnit.Framework;

namespace ProcessCsharpTests {

    [TestFixture()]
    public class ProcessCsharpTest {

        [Test()]
        public void ShouldWork() {
            Assert.AreEqual(ProcessCsharp.Process("Name", "../../test_data"), ProcessCsharp.SUCCESS);
        }
    }
}
